scripts/build.ps1

```diff
@@ -166,6 +169,9 @@ param (
     [Parameter(Mandatory = $false)]
     [switch]$PGO = $false,
 
+    [Parameter(Mandatory = $false)]
+    [switch]$EnableXDP = $false,
```

Comment:
XDP is **always** supported on Windows. This flag is specific to Linux. Let's calls this `EnableLinuxXDP`. Similarly, let's call the cmake flag `QUIC_LINUX_XDP_ENABLED`.

Reply:
...and, this eliminates the need for the separate `QUIC_CARGO_BUILD` flag. Please remove that and fix up the existing usage.

Finally, we need to actually use the `EnableLinuxXDP` flag in our CI jobs.